code/datums/jobs.dm

```diff
@@ -731,22 +765,22 @@
 				return
 			M.show_text("<b>Something has gone terribly wrong here! Search for survivors and escape together.</b>", "blue")
 
-// Civilian Jobs
+// Catering Jobs
 
-/datum/job/civilian
-	linkcolor = "#0099FF"
-	slot_card = /obj/item/card/id/civilian
+/datum/job/catering
+	linkcolor = "#3c45de"
+	slot_card = /obj/item/card/id/catering
 
-/datum/job/civilian/chef
+/datum/job/catering/chef
 	name = "Chef"
-	limit = 1
+	limit = 2
```

Comment:
Do we need a second chef if we're also adding a Head of Catering who can act as a chef?

Reply:
HoC can also access hydro and bar, the idea is that the chefs can collaborate on bigger and stupider projects in the same way robotics starts with 3 max roboticists

Reply:
Why not utilize the existing sous-chef job?

Reply:
Why not utilize the existing sous-chef job?

Reply:
Why not utilize the existing sous-chef job?

Reply:
I'll do that now actually, I didn't want to spam the job interface with more roles